presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,26 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that the Presto process runs as. Currently, 
```

Comment:
See if you can avoid ending the sentence in a preposition

Reply:
`adequate ulimits for the user that runs the Presto process`